package.json

```
@@ -7,7 +7,7 @@
     "commitmsg": "secret-squirrel-commitmsg",
     "precommit": "secret-squirrel",
     "start": "node server.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "mocha test/server.test.js"
```

💯

---

```suggestion
    "test": "mocha test/server.test.js --exit"
```
See [Command Line Usage](https://mochajs.org/#command-line-usage)
> --exit                     Force Mocha to quit after tests complete

This might be handy to have, in case tests hang on the CI?

---

Yeah I was curious about that

---

amazing, thanks